[2024-11-01T06:22:34.867+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-01T06:22:34.880+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: self-tutorial-dag.echo_input manual__2024-11-01T06:22:31.654136+00:00 [queued]>
[2024-11-01T06:22:34.886+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: self-tutorial-dag.echo_input manual__2024-11-01T06:22:31.654136+00:00 [queued]>
[2024-11-01T06:22:34.886+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-11-01T06:22:34.896+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): echo_input> on 2024-11-01 06:22:31.654136+00:00
[2024-11-01T06:22:34.903+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2750) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-01T06:22:34.904+0000] {standard_task_runner.py:72} INFO - Started process 2751 to run task
[2024-11-01T06:22:34.904+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'self-tutorial-dag', 'echo_input', 'manual__2024-11-01T06:22:31.654136+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/self-tutorial-dag.py', '--cfg-path', '/tmp/tmpsquz6uc_']
[2024-11-01T06:22:34.905+0000] {standard_task_runner.py:105} INFO - Job 57: Subtask echo_input
[2024-11-01T06:22:34.944+0000] {task_command.py:467} INFO - Running <TaskInstance: self-tutorial-dag.echo_input manual__2024-11-01T06:22:31.654136+00:00 [running]> on host 2839f63d218a
[2024-11-01T06:22:35.013+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='self-tutorial-dag' AIRFLOW_CTX_TASK_ID='echo_input' AIRFLOW_CTX_EXECUTION_DATE='2024-11-01T06:22:31.654136+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-01T06:22:31.654136+00:00'
[2024-11-01T06:22:35.013+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-01T06:22:35.034+0000] {logging_mixin.py:190} INFO - ##################################################About to fall a sleep
[2024-11-01T06:22:35.035+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-01T06:22:35.042+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-01T06:22:35.042+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=self-tutorial-dag, task_id=echo_input, run_id=manual__2024-11-01T06:22:31.654136+00:00, execution_date=20241101T062231, start_date=20241101T062234, end_date=20241101T062235
[2024-11-01T06:22:35.079+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-01T06:22:35.093+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-01T06:22:35.095+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
